contextvh/src/main/java/contextvh/translators/J2UpgradeType.java

```diff
@@ -18,6 +18,13 @@
 	private static Translator translator = Translator.getInstance();
 
 	/**
+	 * Adds a translator for J2UpgradePair if it isn't added yet.
+	 */
+	public J2UpgradeType() {
+		translator.registerJava2ParameterTranslator(new J2UpgradePair());
```

Comment:
Please verify this is necessary, I haven't seen this in other Translator classes that use other Translators, but for some reason the test for this class was failing.

`tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8`